inference-engine/tests/ngraph_functions/src/detection_output.cpp

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "ngraph_functions/builders.hpp"
+
+namespace ngraph {
+namespace builder {
+
+std::shared_ptr<ngraph::Node> makeDetectionOutput(const ngraph::OutputVector &inputs,
+                                                  const ngraph::op::DetectionOutputAttrs& attrs) {
+    if (inputs.size() == 3)
+        return std::make_shared<ngraph::opset3::DetectionOutput>(inputs[0], inputs[1], inputs[2], attrs);
+    else
+        return std::make_shared<ngraph::opset3::DetectionOutput>(inputs[0], inputs[1], inputs[2], inputs[3], inputs[4], attrs);
+}
+
+}  // namespace builder
+}  // namespace ngraph
```

Comment:
And general comment, Where is ngraph reference implementation?

Reply:
Move reference implementation to Ngraph test in order not to increase Ngraph binary size.